### PR TITLE
Homework16

### PR DESCRIPTION
**Pull request by LidiiaSinikova**
HW # 16 has been done.

### CODE REVIEW

**Comment by Berichicko** [comment]
Hello 
Your homework 15 is checked
**You’re doing great! Keep it up! Very fine!** 
💖Best wishes, the team HW (Alex,  aliaksandr@testpro.io)!💖

giphy


image

**Comment by LidiiaSinikova** [comment]
> Hello Your homework 15 is checked **You’re doing great! Keep it up! Very fine!** 💖Best wishes, the team HW (Alex, [aliaksandr@testpro.io](mailto:aliaksandr@testpro.io))!💖
> 
> giphy [ giphy ](https://user-images.githubusercontent.com/104761482/224489514-d22b527b-705f-4444-ab44-b5d0fded7d97.gif) [ ](https://user-images.githubusercontent.com/104761482/224489514-d22b527b-705f-4444-ab44-b5d0fded7d97.gif)
> 
> image

`1

**Comment by LidiiaSinikova** [comment]
I'm sorry, it was HW # 16. i sent a link there..

File: src/test/java/Homework16.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public void Navigation() {
```
Comment (Berichicko) [inline src/test/java/Homework16.java:13]: Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example public void navigation()

**Comment by Berichicko** [comment]
Please, add Homework16 in TestNG.xml to run tests

image

**Comment by LidiiaSinikova** [comment]
I did, it was not running at all, but Svetlana advised me to do this and test passes, though i had some corrections

File: src/test/java/Homework16.java (hunk 1 above)
Comment (LidiiaSinikova) [reply src/test/java/Homework16.java:13]: I changed it to small navigation() and sent changed on GitHub

Comment (Berichicko) [reply src/test/java/Homework16.java:13]: > I changed it to small navigation() and sent changed on GitHub

Please, re-submit homework 16 for review

**Pull request closed without merging**